nupic/research/frameworks/vernon/mixins/regularize_loss.py

```diff
@@ -80,7 +80,7 @@ def pre_batch(self, batch_idx, **kwargs):
 
     def complexity_loss(self, model):
         c_loss = super().complexity_loss(model)
-        assert model is self.model
+        # assert model is self.model
```

Comment:
Two things:

1. This is the loss function and will be called regularly, so having an `assert` statement will slow down the overall runtime; you should consider removing it.
2. If there's code that you don't need, delete it instead of commenting it out.

Reply:
Ah, you commented out the `assert` statement -- my bad. In that case, only #2 above applies.